packages/opentelemetry-plugin-postgres/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@opentelemetry/plugin-postgres",
```

Comment:
Unsure if this should be named `plugin-postgres` or `plugin-pg`. The actual npm package name is [pg](https://www.npmjs.com/package/pg) and not [postgres](https://www.npmjs.com/package/postgres), so naming it `plugin-pg` would be less confusing. Naming it `plugin-postgres` gives better visibility of what the plugin does.

Reply:
+1 for `plugin-postgres`.

Reply:
+1 for plugin-postgres.